Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
+{
+    protected List<char> _listOfLettersToCount;
+    protected List<char> _listOfNumberToCount;
+    private List<char> _illegalalCharacters;
+    private int? _result;
+
+    public Helper()
+    {
+        _listOfLettersToCount = ['a', 'b', 'c'];
+        _listOfNumberToCount = ['1', '2', '3'];
+        _illegalalCharacters = ['-', '<', '>'];
+    }
+
+    public bool IsCharacterAndIfINeedToCount(char character)
+    {
+        if (char.IsAsciiLetter(character)) 
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public bool IsDigitAndIfItIsInTheList(char character)
+    {
+        if (char.IsDigit(character))
+        {
+            var letter = character;
+
+            if (_listOfLettersToCount.Contains(letter))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        return false;
+    }
+
+    public void IncreaseDigitCount(int digitCount, bool b)
+    {
+        if (b)
```

Comment:
Manau vertėtų pakeisti funckiją ir returninti ++digitCount (atitinkamai void -> int), dar gali pridėti else statment, kuriame grąžini digitCount

Reply:
Kodėl ?

Reply:
Jeigu klausiama dėl ++digitCount, tai čia pirmiausia bus padidinama vertė ir tik po to vykdomas veiksmas, pakeisti iš void į int, nes returninsime digitCount, else grąžiname tiesiog digitCount, nes paduodama bool vertė teigia, kad keisti vertės nereikia.

Reply:
What is the difference between value types and reference types ?